tests/test-cmd/test.build.js

```diff
@@ -209,7 +209,7 @@ describe('build', () => {
       assert.equal(filter.wantFile('some.xpi'), true);
       assert.equal(filter.wantFile('manifest.json'), false);
     });
-    
+
```

Comment:
And my editor just deletes trailing whitespace so I didn't even notice I committed it 😄